openshift/e2e-tests-openshift.sh

```diff
@@ -52,12 +49,12 @@ function install_strimzi(){
 function install_knative_serving(){
   header "Installing Knative Serving"
 
-  create_knative_namespace serving
+  # create_knative_namespace serving
```

Comment:
May be just delete this line?

Reply:
Nope, needed with TP1 - this is to get the 0.8.0 build going ;-)